platforms/documentation/docs/src/docs/userguide/authoring-builds/build_environment.adoc

```diff
@@ -87,65 +87,92 @@ Here are all possible configurations of specifying the JDK installation director
 +
 [source,text]
 ----
-$ ./gradlew yourTask -Dorg.gradle.java.home=/path/to/your/java/home
+$ ./gradlew exampleTask -Dorg.gradle.java.home=/path/to/your/java/home --scan
 ----
 2. Gradle Properties File
 +
 [source,text]
+.gradle.properties
 ----
 org.gradle.java.home=/path/to/your/java/home
+systemProp.http.proxyHost=host.org
 ----
 3. Environment Variable
 +
 [source,text]
 ----
 $ export JAVA_HOME=/path/to/your/java/home
+$ export JAVA_OPTS="-Xmx18928m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8 -Djava.awt.headless=true -Dkotlin.daemon.jvm.options=-Xmx6309m"
+----
+
+== The `gradle.properties` file
+
+Gradle properties, system properties, and project properties can be found in the `gradle.properties` file:
+
+[source, properties]
+.gradle.properties
+----
+# Gradle properties
+org.gradle.parallel=true
+org.gradle.caching=true
+org.gradle.jvmargs=-Duser.language=en -Duser.country=US -Dfile.encoding=UTF-8
+
+# System properties
+systemProp.pts.enabled=true
+systemProp.log4j2.disableJmx=true
+systemProp.file.encoding = UTF-8
+
+# Project properties
+kotlin.code.style=official
+android.nonTransitiveRClass=false
+spring-boot.version = 2.2.1.RELEASE
 ----
 
 [[sec:project_properties]]
 == Project properties
 
 Project properties are specific to your Gradle project, they are variables or configuration blocks defined in your build script or in the `project` object.
 
-For example, the project property `myProperty` is created in a build file:
+For example, the project property `dryRun`, in the build file below, enables a specific test mode:
```

Comment:
Something went wrong with this example because `dryRun` below is just an extension property. The example doesn't actually demonstrate a project property at all.